XML/TestCases/FunctionalTests-GPU.xml

```diff
@@ -27,43 +27,30 @@
         <Priority>0</Priority>
     </test>
     <test>
-        <testName>NVIDIA-CUDA-DRIVER-TENSORFLOW</testName>
-        <preTestScript>GPU-DRIVER-INSTALL.ps1</preTestScript>
-        <testScript>GPU-TENSORFLOW.ps1</testScript>
+        <testName>NVIDIA-CUDA-DRIVER-VALIDATION-MAX-GPU</testName>
+        <testScript>GPU-DRIVER-INSTALL.ps1</testScript>
         <setupType>OneVM</setupType>
-        <OverrideVMSize>Standard_NC6</OverrideVMSize>
-        <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\gpu-driver-install.sh,.\Testscripts\Linux\gpu-tensorflow.sh</files>
+        <OverrideVMSize>Standard_NC24</OverrideVMSize>
+        <files>.\Testscripts\Linux\gpu-driver-install.sh,.\Testscripts\Linux\utils.sh</files>
         <TestParameters>
             <param>CUDADriverVersion="CUDA_DRIVER"</param>
-            <param>CudaToolkitVersion="CUDATOOLKIT_VERSION"</param>
-            <param>TensorflowVersion="TENSORFLOW_VERSION"</param>
-            <param>CudnnPackage="CUDNN_LINUX_PACKAGE"</param>
-            <param>BenchmarkTool="TENSORFLOW_BENCHMARK"</param>
         </TestParameters>
         <Platform>Azure</Platform>
         <Category>Functional</Category>
         <Area>GPU</Area>
-        <Tags>gpu,benchmark</Tags>
+        <Tags>gpu,pci_hyperv</Tags>
         <Priority>1</Priority>
     </test>
     <test>
-        <testName>NVIDIA-GRID-DRIVER-TENSORFLOW</testName>
-        <preTestScript>GPU-DRIVER-INSTALL.ps1</preTestScript>
-        <testScript>GPU-TENSORFLOW.ps1</testScript>
+        <testName>NVIDIA-GRID-DRIVER-VALIDATION-MAX-GPU</testName>
+        <testScript>GPU-DRIVER-INSTALL.ps1</testScript>
         <setupType>OneVM</setupType>
-        <OverrideVMSize>Standard_NV6</OverrideVMSize>
-        <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\gpu-driver-install.sh,.\Testscripts\Linux\gpu-tensorflow.sh</files>
-        <TestParameters>
-            <param>CUDADriverVersion="CUDA_DRIVER"</param>
```

Comment:
here too? they are removed. We'd like to control each parameter in Custom Parameters.